### DISCUSSION TITLE
Anonymous sign-in support

### DISCUSSION THREAD

**Discussion by orenaksakal** [pocketbase/pocketbase#4684] [Q&A] [answered]
Hello 👋 just curious if there are any plans, implementations or ideas I can peak to support anonymous sessions like [supabase](https://supabase.com/docs/guides/auth/auth-anonymous)?

**Comment by ganigeorgiev** [comment, marked as answer]
There is no support for anonymous sign-in, at least not directly. You could check https://github.com/pocketbase/pocketbase/discussions/3035 and other similar posts.
